src/fides/api/api/v1/endpoints/served_notice_endpoints.py

```diff
@@ -76,14 +76,18 @@ def save_consent_served_for_identities(
     common_data["serving_component"] = original_request_data.serving_component
 
     def save_consent_served_for_field_name(
-        identifiers: Union[List[SafeStr], List[int]], field_name: ConsentRecordType
+        identifiers: Optional[Union[List[SafeStr], List[int]]],
+        field_name: Optional[ConsentRecordType],
     ) -> None:
         """Internal helper for creating a ServedNoticeHistory record for various types
         of consent components
 
         Loops through the list of all consent components of a given type and saves
         to the database that they were served.
         """
+        if not field_name or not identifiers:
+            return
+
```

Comment:
This just lets me skip saving that consent was served for the "vendor relationships" section which is the only one that is read-only -